charts/engine/templates/engine-job-config.yaml

```
@@ -0,0 +1,20 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: engine-job-config
+data:
+{{ if .Values.gpu }}
+  defaultGpuType: engine-job-template-k80-cm
+{{ else }}
+  defaultGpuType: engine-job-template-pysvcs-cm
+{{ end }}
+  defaultCpuType: engine-job-template-pysvcs-cm
+  machineTypes: |
+    - id: cpu
+      displayName: CPU
+      templateConfigMap: engine-job-template-pysvcs-cm
+{{ if .Values.gpu }}
+    - id: k80
+      displayName: GPU
+      templateConfigMap: engine-job-template-k80-cm
```

I think it's a good time to _find and replace in all files_ `k80` to `gpu`
```suggestion
      templateConfigMap: engine-job-template-gpu-cm
```